docker/images/pinot/etc/jmx_prometheus_javaagent/configs/broker.yml

```diff
@@ -1,76 +1,90 @@
 rules:
-- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.([^\\.]*?).authorization\"><>(\\w+)"
-  name: "pinot_broker_authorization_$2"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.(([^\\.]+?)#)*([^\\.]*?).authorization\"><>(\\w+)"
+  name: "pinot_broker_authorization_$4"
   cache: true
   labels:
-    table: "$1"
-- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.([^\\.]*?)\\.documentsScanned\"><>(\\w+)"
-  name: "pinot_broker_documentsScanned_$2"
+    database: "$2"
```

Comment:
I think database is a logical concept, can we keep it as part of table name? If we want to filter database, we can do it on the metric query side.

Reply:
Makes sense. While keeping it as part of the table let's also emit it as a label separately to ease the filtering maybe?